scripts/parks/ParkPreview.js

```diff
@@ -0,0 +1,41 @@
+import { useParks } from "./ParkProvider.js"
+
+const eventHub = document.querySelector(".container")
+const contentTarget = document.querySelector(".preview--park")
+
+contentTarget.addEventListener("click", event => {
+  if(event.target.id.startsWith("openParkDetail--") || event.target.id.startsWith("closeParkDetail--")) {
+    const [ prefix, parkId ] = event.target.id.split("--")
+    const dialogNode = document.querySelector(`#parkDialog--${parkId}`)
+
+    if(prefix === "openParkDetail") {
+      dialogNode.showModal()
+    }
+    else {
+      dialogNode.close()
+    }
+  }
```

Comment:
Stylistic preference, but I find that the way we implemented the event listener in the oddity preview is easier to understand/modify.

Reply:
Cool, I'll change it. The only reason I wrote it the way I did here is cause the other way relies on the close button being a direct child of the `<dialog>` HTML element, and if that button was ever further nested say within a `<div>` or something, it would break. BUT if we just make sure to not change our HTML too much in the `<dialog>` that way should work just fine.

Reply:
Lol never mind the OddityPreview way does not rely on using parentNode inherently, basically ignore my previous comment.

Reply:
OK I wrote it closer to how OddityPreview implemented it... let me know if the way I did it this time is okay.

Reply:
Looks good!